nbactions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Depends on the project, of course, but typically IDE related files are undesired.